app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -143,6 +155,10 @@ def to_enc
       }.merge(period_enc)
     end
 
+    def find_assets(method)
```

Comment:
not quite, I use it to call scopes on assets comfortably, `policy.find_assets(:incomply_with)` is really `Scaptimony::Asset.incomply_with(policy)`